schema/codelists/contractFinalStatus.csv

```diff
@@ -0,0 +1,5 @@
+Code,Title,Description
+terminatedSuccessfully,Terminated successfully,"The contract was concluded and in force, and is terminated due to its successful completion."
+terminatedEarly,Terminated early,"The contract was concluded and in force, and is terminated early due to some non-completion."
+cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the award was made and the standstill period, if any, had ended."
```

Comment:
```suggestion
cancelled,Cancelled,"The contracting process was cancelled by the buyer or the procuring entity (for example, because of a change in needs, insufficient funds, or technical or procedural errors) after the award was made and the standstill period, if any, had ended."
```